src/org.openSUSE.YaST.appdata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id>org.openSUSE.YaST</id>
+  <launchable type="desktop-id">org.openSUSE.YaST.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
is it ok from legal perspective to not mention CC in rpm spec when we also distrubute it?

Reply:
Good question, in some places CC0 is a way around lack of laws regarding public domain, it would probably be better to license it the same as the code itself